e2e_tests/tests/fixtures/configuration/run.py

```diff
@@ -0,0 +1,9 @@
+import time
```

Comment:
could we please rename this `configuration` -> `generic_task` to make it clearer which feature this fixture refers to?